src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -506,4 +506,28 @@ describe('Users edit mode', () => {
     const expectedRequest = usersCreateBlueprintRequest;
     expect(receivedRequest).toEqual(expectedRequest);
   });
+
+  test('shows tooltip on disabled password eye icon when editing blueprint with existing password', async () => {
+    const id = mockBlueprintIds['users'];
+    await renderEditMode(id);
+
+    await clickRevisitButton();
+    
+    const expandable = await screen.findByTestId('users-expandable');
+    const allButtons = await within(expandable).findAllByRole('button');
+    
+    const passwordButtons = allButtons.filter(
+      (button) => button.getAttribute('aria-label')?.toLowerCase().includes('password') ?? false
+    );
+    
+    expect(passwordButtons).toHaveLength(0);
+    
+    const allElements = await within(expandable).findAllByRole('button');
+    const passwordElements = allElements.filter(element => 
+      element.getAttribute('aria-label')?.toLowerCase().includes('password') ||
+      element.textContent?.toLowerCase().includes('password')
+    );
+  
+    expect(passwordElements).toHaveLength(0);
+  });
```

Comment:
Got lost in the test, it doesn't really do what it says in the name of it.

There are no buttons in the users expandable on the review step so I don't think we need to check if there are or their status.

What would make sense to me is:
- render edit mode
- go to the Users step
- hover on the password visibility button
- check for the tooltip
- unhover
- make sure the tooltip disappears

the test could also double check that the button is indeed disabled, I think that would be nice. What do you think? :)